README.md

```diff
@@ -119,6 +119,28 @@ has_event :complete, field_name: :completion_time
 
 Note that the `field_name` option takes precedence over the `field_type` option.
 
+### Comparison strategy
+
+By default the field's presence will dictate the behavior. However in some cases you may want to check against the current time.
+
+You can do this with the `strategy` option, which can be either `presence` or `time_comparison`:
+
+```ruby
+has_event :complete, strategy: :time_comparison
+```
+
+**Example:**
+
+```ruby
+task.completed_at = 1.hour.ago
+task.completed? # => true
+Task.completed  # => [...]
+
+task.completed_at = 1.hour.from_now
+task.completed? # => false
+Task.completed  # => []
```

Comment:
For the sake of simplicity, I would remove the lines that demonstrate the use of scopes. Also, I think they won't work as expected unless the record is saved first.